code/modules/food_and_drinks/food/foods/desserts.dm

```diff
@@ -30,12 +31,20 @@
 	bitesize = 3
 	list_reagents = list("nutriment" = 3, "sugar" = 7, "ice" = 2)
 
+/obj/item/reagent_containers/food/snacks/icecream/icecreamwafflecone
```

Comment:
could you shorten this typename to `wafflecone` since the parent type is already `icecream`